Submissions/Ekey_GroceryApp/GroceryApp/GroceryStoreTripModel/GSTTableViewCell.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  GSTTableViewCell.swift
+//  GroceryApp
+//
+//  Created by johnekey on 4/1/19.
+//
+
+import UIKit
+
+class GSTTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var nameTextField: UILabel!
+    @IBOutlet weak var ttlCostTextField: UILabel!
+    @IBOutlet weak var costTextField: UILabel!
+    @IBOutlet weak var qtyTextField: UILabel!
+    
+    override func awakeFromNib() {
```

Comment:
what is a nib and why do I have to awaken it??

Reply:
the nib is basically the storyboard. And you don't awaken it, you awake from it. :p